help_to_heat/settings.py

```diff
@@ -253,9 +253,8 @@ def show_toolbar(request):
 }
 
 OS_API_KEY = env.str("OS_API_KEY")
-EPC_API_BASE_URL = env.str("EPC_API_BASE_URL")
-EPC_API_CLIENT_ID = env.str("EPC_API_CLIENT_ID")
-EPC_API_CLIENT_SECRET = env.str("EPC_API_CLIENT_SECRET")
```

Comment:
Were these being used somewhere in the app? I'd expect them to be consumed somewhere we're actually using the API

Reply:
Ah I wasn't sure that was part of the scope. I've made some further changes now.
I'm going to sort the tests, but could you give the code a skim? (understand time is tight today)

Reply:
Might not have been, but I suppose the auth part is useful to have, and it's only going to save future effort for the other bits. If it turns out to be a load of work or you're getting too close to actually doing the implementation then I suppose you could leave the current implementation as-is and have the new stuff commented out to be used on the implementation tickets?